docs/transpile/customizing-transpilation.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>oh I realise now there is a referene for dd haha. Could you move this up to the intro paragraph where you first mention it? I think it would also be good to put it in and admonition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>